rfhub2/cli/rfhub_importer.py

```diff
@@ -0,0 +1,216 @@
+        pointing to libraries to import to app.
+        :return: Set of Paths object pointing to libraries to import
+        """
+        self.client.check_communication_with_app()
+        libraries_paths = set()
+        for path in self.paths:
+            libraries_paths.update(self._traverse_paths(Path(path)))
+        if not self.no_installed_keywords:
+            libdir = Path(robot.libraries.__file__).parent
+            libraries_paths.update(self._traverse_paths(Path(libdir)))
+        return libraries_paths
+
+    def _traverse_paths(self, path: Path) -> Set[Path]:
+        """
+        Traverses through paths and adds libraries to rfhub.
+        Helper function for get_library_paths.
+        """
+        valid_lib_paths = set()
+        if self._is_library_with_init(path):
+            valid_lib_paths.add(path)
+        else:
+            for item in path.iterdir():
+                if item.is_dir():
+                    if self._is_library_with_init(item):
+                        valid_lib_paths.add(item)
+                    else:
+                        valid_lib_paths.update(self._traverse_paths(item))
+                elif item.is_file() and self._is_robot_keyword_file(item) and not self._should_ignore(item):
+                    valid_lib_paths.add(item)
+        return valid_lib_paths
+
+    def create_collections(self, paths: Set[Path]) -> List[Collection]:
+        """
+        Creates list of Collection objects from set of provided paths.
+        :param paths: set of paths
+        :return: list of Collection objects
+        """
+        return [self.create_collection(path) for path in paths]
+        # ToDo try except to handle problem with library creation
+
+    def create_collection(self, path: Path) -> Collection:
+        """
+        Creates Collection object from provided path.
+        :param path: Path
+        :return: Collection object
+        """
+        libdoc = LibraryDocumentation(str(path))
+        serialised_keywords = self._serialise_keywords(libdoc)
+        return self._serialise_libdoc(libdoc, str(path), serialised_keywords)
+
+    def add_collections(self, collections: List[Collection]) -> List[Dict[str, int]]:
+        """
+        Adds collections and keywords from provided list to app.
+        :param collections: List of collections object
+        :return: list of dictionaries with collection name and number of keywords.
+        """
+        loaded_collections = []
+        for collection in collections:
+            coll_req = self.client.add_collection(collection)
+            if coll_req['name'] == collection['name']:
+                collection_id = coll_req['id']
+                for keyword in collection['keywords']:
+                    keyword['collection_id'] = collection_id
+                    self.client.add_keyword(keyword)
+                loaded_collections.append({collection['name']: len(collection["keywords"])})
```

Comment:
How about changing the dict to: `{'name': collection['name'], 'keywords': len(collection["keywords"])}` to have more object-like dicts with well defined keys?

Reply:
Yeah, no problem. Fixed